bitnami/clickhouse-operator/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+CHART NAME: {{ .Chart.Name }}
+CHART VERSION: {{ .Chart.Version }}
+APP VERSION: {{ .Chart.AppVersion }}
+
+Did you know there are enterprise versions of the Bitnami catalog? For enhanced secure software supply chain features, unlimited pulls from Docker, LTS support, or application customization, see Bitnami Premium or Tanzu Application Catalog. See https://www.arrow.com/globalecs/na/vendors/bitnami for more information.
+
+** Please be patient while the chart is being deployed **
+
+Check the status of the pods by running this command:
+
+  kubectl get pods --namespace {{ include "common.names.namespace" . | quote }} -l app.kubernetes.io/instance={{ .Release.Name }}
+
+Check the ClickHouse Operator documentation for instructions on how to deploy a ClickHouse cluster:
+
+  https://docs.altinity.com/altinitykubernetesoperator/kubernetesquickstartguide/quickcluster
+
```

Comment:
I would add a warning in the notes if the operator can see secrets in all namespaces. See cloudnative-pg

Reply:
done at https://github.com/bitnami/charts/pull/32816/commits/f07d8c5a0c14c4e10078b83f623887f6736a00ff